specification/cognitiveservices/data-plane/ComputerVision/preview/v3.0/Ocr.json

```diff
@@ -0,0 +1,447 @@
+        "parameters": [
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Id of read operation returned in the response of the 'Read' interface.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the read operation status.",
+            "schema": {
+              "$ref": "#/definitions/ReadOperationResult"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Read Result request": {
+            "$ref": "./examples/SuccessfulGetReadResult.json"
+          }
+        }
+      }
+    }
+  },
+  "x-ms-paths": {
+    "/read/analyze?overload=stream": {
+      "post": {
+        "description": "Use this interface to get the result of a Read operation, employing the state-of-the-art Optical Character Recognition (OCR) algorithms optimized for text-heavy documents. When you use the Read interface, the response contains a field called 'Operation-Location'. The 'Operation-Location' field contains the URL that you must use for your 'GetReadResult' operation to access OCR results.​",
+        "operationId": "ReadInStream",
+        "parameters": [
+          {
+            "$ref": "#/parameters/OcrDetectionLanguage"
+          },
+          {
+            "$ref": "#/parameters/ImageStream"
+          }
+        ],
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation. The operation ID will expire in 48 hours. ",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Analyze request": {
+            "$ref": "./examples/SuccessfulReadWithStream.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ReadOperationResult": {
+      "type": "object",
+      "description": "OCR result of the read operation.",
+      "properties": {
+        "status": {
+          "description": "Status of the read operation.",
+          "$ref": "#/definitions/OperationStatus"
+        },
+        "createdDateTime": {
+          "type": "string",
```

Comment:
You could also add "format" if your date format is consistent (always ISO8601) for instance:
- `format:date-time` means ISO8601 (full date + hours)
- `format:date` means ISO8601 (full date + hours)
- `format:date-time-rfc1123` means RFC1123